### PR TITLE
Library icon improvements

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Before:

We said "(test)" at the end of a team name

Now:

* Teams get a folder icon
* Tests get their own icon

image

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
cc @jaril or @ryanjduffy

**Comment by jaril** [comment]
> cc @jaril or @ryanjduffy

I'd probably split the icon out into its own component. Then from there, you can early return with the testsuite icon if `isTest === true`. 

Distinguishing the user's library vs a team is pretty tricky, but if you copy the pattern here, that should be able to let you differentiate between the two.

https://github.com/replayio/devtools/blob/ef3b51f1045bf7f423b14338d611d911f92e3df9/src/ui/components/Library/Library.tsx#L45

File: src/ui/components/Library/Navigation/TeamButton.tsx [hunk 1]
```diff
@@ -71,6 +66,12 @@ export function TeamButton({
   );
 }
 
+export function LibraryIcon(teamType: string) {
```
Comment (jaril) [inline src/ui/components/Library/Navigation/TeamButton.tsx:66]: This should be `export function LibraryIcon({ teamType }: { teamType: string })`

File: src/ui/components/Library/Navigation/TeamButton.tsx [hunk 2]
```diff
@@ -49,18 +51,11 @@ export function TeamButton({
...
-        (  
-          <Icon filename="other" size="medium" className="mr-1 bg-iconColor" />
-        ) 
-        : 
-          <Icon filename="tests" size="medium" className="mr-1 bg-iconColor" />
-        }
-
+        <span className="overflow-hidden whitespace-pre overflow-ellipsis">        
+        <div className="flex">        
+        <LibraryIcon teamType={isTest ? ("tests") : ("team")} />
```
Comment (jaril) [inline src/ui/components/Library/Navigation/TeamButton.tsx]: Can simplify this to `<LibraryIcon teamType={isTest ? "tests" : "team"} />  `

**Review by jaril — CHANGES_REQUESTED** [review]

File: src/ui/components/Library/Navigation/TeamButton.tsx (hunk 1 above)
Comment (jonbell-lot23) [reply src/ui/components/Library/Navigation/TeamButton.tsx:66]: Fixed!

File: src/ui/components/Library/Navigation/TeamButton.tsx (hunk 2 above)
Comment (jonbell-lot23) [reply src/ui/components/Library/Navigation/TeamButton.tsx]: Fixed!

**Comment by jonbell-lot23** [comment]
Hey @jaril I addressed your requests from a week ago, let me know what you think

**Pull request merged**